cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -398,6 +398,10 @@ spec:
             {{- else }}
               value: production
             {{- end }}
+            {{- if .Values.kubecostProductConfigs.productKey }}
+            - name: PRODUCT_KEY_MOUNT_PATH
+              value: {{ .Values.kubecostProductConfigs.productKey.envvarMountPath | default "/var/configs/productkey" }}
```

Comment:
Open question - thoughts on the clarity of the names `volumeMountPath` and `envvarMountPath`?